mod_reforged/hooks/config/strings.nut

```diff
@@ -1898,18 +1886,14 @@ foreach (vanillaDesc in vanillaDescriptions)
  	}),
 	RF_Tempo = ::UPD.getDescription({
  		Fluff = "By keeping ahead of your opponent, you set the terms of engagement!",
+ 		Requirement = "Sword",
  		Effects = [
 	 		{
 				Type = ::UPD.EffectType.Passive,
 				Description = [
-					"Every attack, hit or miss, against a target who acts after you in the current round increases your [Initiative|Concept.Initiative] by " + ::MSU.Text.colorGreen("+15") + ".",
-					"The bonus lasts over into your next [turn|Concept.Turn] but only until the first skill used or waiting that [turn|Concept.Turn]."
-				]
-			},
-			{
-				Type = ::UPD.EffectType.Passive,
-				Description = [
-					"When wielding a sword, gain the [Fluid Weapon|NullEntitySkill+rf_fluid_weapon_effect] effect."
+					"Your [Initiative|Concept.Initiative] is considered " + ::MSU.Text.colorGreen("50%") + " higher for calculating your [turn order|Concept.Turn].",
+					"Every hit or miss against a target who acts after you in the current round increases your [Initiative|Concept.Initiative] by " + ::MSU.Text.colorGreen("+15") + ". This bonus lasts over into your next [turn|Concept.Turn] but only until the first skill used or waiting that [turn|Concept.Turn].",
+					"The first two hits against opponents who act after you in the current [round|Concept.Round] recover " + ::MSU.Text.colorGreen("2") + " and " + ::MSU.Text.colorGreen("1") + " [Action Points|Concept.ActionPoints] respectively. Misses increment the counter, but do not recover any [Action Points|Concept.ActionPoints].",
```

Comment:
The part where it first recovers 2 AP and then 1 AP is complicated in description and in code for barely any gain as I pointed out to you in discord. I would remove that part and make both attacks restore 2 AP.

I only remember you reply with some calculations but nothing else.
Have you reached a verdict about that?